tests/scipy_optimize_test.py

```diff
@@ -102,5 +102,95 @@ def f(x):
       jax.scipy.optimize.minimize(f, jnp.ones(2), args=45, method='BFGS')
 
 
+class TestLBFGS(jtu.JaxTestCase):
+
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "_func={}_maxiter={}".format(func_and_init[0].__name__, maxiter),
+     "maxiter": maxiter, "func_and_init": func_and_init}
+    for maxiter in [None]
+    for func_and_init in [(rosenbrock, np.zeros(2)),
+                          (himmelblau, np.zeros(2)),
+                          (matyas, np.ones(2) * 6.),
+                          (eggholder, np.ones(2) * 100.)]))
+  def test_minimize(self, maxiter, func_and_init):
+
+    func, x0 = func_and_init
+
+    @jit
+    def min_op(x0):
+      result = jax.scipy.optimize.minimize(
+          func(jnp),
+          x0,
+          method='L-BFGS',
```

Comment:
the method name needs to be updated here and elsewhere in the tests, too :)

Reply:
Sorry to let you do this for me,
I forgot about the push completely.
Fix incoming